server/src/main/java/com/objectcomputing/checkins/services/feedback_request/FeedbackRequestController.java

```diff
@@ -0,0 +1,117 @@
+    private final EventLoopGroup eventLoopGroup;
+    private final ExecutorService executorService;
+    private final CurrentUserServices currentUserServices;
+
+    public FeedbackRequestController(FeedbackRequestServices feedbackReqServices,
+                                     EventLoopGroup eventLoopGroup,
+                                     @Named(TaskExecutors.IO) ExecutorService executorService,
+                                     CurrentUserServices currentUserServices) {
+        this.feedbackReqServices = feedbackReqServices;
+        this.eventLoopGroup = eventLoopGroup;
+        this.executorService = executorService;
+        this.currentUserServices = currentUserServices;
+    }
+
+    /**
+     * Create a feedback request
+     *
+     * @param requestBody {@link FeedbackRequestCreateDTO} New feedback request to create
+     * @return {@link FeedbackResponseDTO}
+     */
+    @Post()
+    public Single<HttpResponse<FeedbackRequestResponseDTO>> save(@Body @Valid @NotNull FeedbackRequestCreateDTO requestBody) {
+        return Single.fromCallable(() -> feedbackReqServices.save(fromDTO(requestBody)))
+                .observeOn(Schedulers.from(eventLoopGroup))
+                .map(savedFeedbackRequest -> (HttpResponse<FeedbackRequestResponseDTO>) HttpResponse
+                        .created(fromEntity(savedFeedbackRequest))
+                        .headers(headers -> headers.location(URI.create("/feedback_request" + savedFeedbackRequest.getId()))))
+                .subscribeOn(Schedulers.from(executorService));
+    }
+
+
+    @Delete("/{id}")
+    public Single<HttpResponse> delete(@NotNull UUID id) {
+        return Single.fromCallable(() -> feedbackReqServices.delete(id))
+                .observeOn(Schedulers.from(eventLoopGroup))
+                .map(successFlag -> (HttpResponse) HttpResponse.ok())
+                .subscribeOn(Schedulers.from(executorService));
+    }
+
+    /**
+     * Get feedback requst by ID
+     *
+     * @param id {@link UUID} ID of the request
+     * @return {@link FeedbackRequestResponseDTO}
+     */
+    @Get("/{id}")
+    public Single<HttpResponse<FeedbackRequestResponseDTO>> getById(UUID id) {
+        return Single.fromCallable(() -> feedbackReqServices.getById(id))
+                .observeOn(Schedulers.from(eventLoopGroup))
+                .map(savedFeedbackRequest -> (HttpResponse<FeedbackRequestResponseDTO>) HttpResponse
+                        .ok(fromEntity(savedFeedbackRequest))
+                        .headers(headers -> headers.location(URI.create("/feedback_request" + savedFeedbackRequest.getId()))))
+                .subscribeOn(Schedulers.from(executorService));
+    }
+    /**
+     * Get feedback request by creator's ID
+     *
+     * @param creatorId {@link UUID} ID of member profile who created the feedback request
+     * @return {@link Set <FeedbackResponseDTO>} Set of feedback requests that were made by certain creator
+     */
+    @Get("/{?creatorId}")
```

Comment:
do we need a get call for `requesteeId` as well ?

Reply:
We can't think of a use case where we would only want to see feedback requests on a specific person (at least not by ID). We would want to see feedback requests filtered by the *name* of the requestee though.